environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -878,6 +898,220 @@ objects:
         - name: telemeter-server-tls
           secret:
             secretName: telemeter-server-shared
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  rules:
+  - nonResourceURLs:
+    - /metrics
+    verbs:
+    - get
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: prometheus-ams-${NAMESPACE}
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-ams
+    namespace: ${NAMESPACE}
+- apiVersion: v1
+  data:
+    nginx.conf: |
+      daemon off;
+      worker_processes 1;
```

Comment:
Why only 1? (I have no background around this, just asking)

Reply:
Because it runs in a container, it spawns a single process which runs on foreground, with 1024 workers. If it fails then container exits.

Reply:
I have checked this again, `nginx` spawns 1 master process and `n` worker processes, best practice is to spawn 1 worker process per core, so what should it be?